basxconnect/core/layouts/editperson.py

```diff
@@ -155,12 +160,18 @@ def editperson_head(request, isreadview):
             style="padding-top: 1rem",
         ),
         R(
-            C(active_toggle(isreadview), width=1, breakpoint="md"),
-            C(personnumber, width=1, breakpoint="md"),
-            C(personmaintype, width=1, breakpoint="md"),
-            C(created, width=1, breakpoint="md"),
-            C(last_change(), width=1, breakpoint="md"),
-            C(),
+            C(
+                hg.DIV(
+                    active_toggle(isreadview),
+                    personnumber,
+                    personmaintype,
+                    created,
+                    last_change(),
+                    style="display:flex;",
```

Comment:
Flex, the solution to so many CSS-problem :) :heart: 
(Except css-grid is required ;)

Reply:
yes, it's really cool for one-dimensional alignment!